### ISSUE TITLE
IA intake from Core Identity Team

### ISSUE THREAD

**Issue by claytonzook** [department-of-veterans-affairs/va.gov-team#62354]
## About your team

- Team name: 
-	OCTO-DE product owner: Tom Black
-	Product manager: Amber Parsley and Mike King
-	Designer: Alex Garcia and Clayton Zook
- Accessibility specialist: none
- Content writer: Tinu Awoliyi
-	Product/team Slack channel: #identity-surge
 
### Describe what you need in a few sentences:
From DI it was suggested that we open a CAIA ticket to

> get help from content to create a new static content page that can support the message we need to communicate to users that click "Cancel." Also request to work with IA to determine where that page will live on Va.gov.

- Bottom line is that we are creating a new page for when folks don't agree to the updated Terms of Use (TOU). This will hold information about the consequences of not agreeing to the updated TOU. The current design of that page:

![Screenshot 2023-07-20 at 1 15 19 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/97549241/899857db-0469-42e4-8c2c-6af3a8a9c605)

### Which are you doing?
- [x] **Launching a new page** (TOU decline page)
- [x] **Making changes** to an existing page (TOU fka [Terms and Conditions](https://www.va.gov/health-care/medical-information-terms-conditions/))

### Types of Support
What areas does your team need support in? Check all that apply.
#### Content
- [ ] Collaboration on content written by your team’s dedicated content writer 
- [ ] Collaboration on content if your team does not have a dedicated content writer
- [x] Content is largely being written by OGC, I believe that sitewide content team (Laura) has already been working with them on content for the updated TOU page. We should be good RE content on the new TOU decline page
#### Accessibility
- [ ] Accessibility review of wireframes and/or prototypes
- [ ] Accessibility issue (general request)
- [ ] Accessibility PDF review request
- [ ] Accessibility user research support ([submit a separate, additional request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name))
- [ ] DST component or pattern 
- [x] Something else - We had an Collab Cycle accessibility review during DI and will be glad to hear additional feedback for MPR, not sure if there is another type of a11y review we should undergo at this point? We don't have any hard code
#### IA
- [ ] User Flows
- [x] Page placement
- [ ] Object-Oriented UX (OOUX) support
- [x] URLs
- [ ] Breadcrumbs and Left Nav placement
- [ ] Redirects

#### Multilingual content
- [ ] Translating content
 
## Timeframe
_We guide and work alongside teams across nearly every OCTO digital product. We also work with partners across VA to lead content migration and manage all unauthenticated content on VA.gov. And we're always working to expand the VA.gov content style guide and our IA and accessibility documentation. We need to prioritize all intake requests based on our overall workload and VA and OCTO priorities._

_Tell us about your product's timeline. And we'll work with you to meet timeline needs as best we can._

### Have you worked with CAIA before? 
  - [x] Yes (content areas)
  - [x] No (IA and a11y areas of CAIA)

### What is your team’s next step and its timing?

- **Our next step is** Midpoint Review AND Research Review - in prep for usability research with veterans
- **Provide date if available**: we hope to have both of those complete by Aug 1st (ish), but still need to schedule each

#### Timing for your next step: 
  - [x] Next week
  - [x] This month
  - [ ] Next month
  - [ ] This quarter
  - [ ] Next quarter
  - [ ] 6 months
  - [ ] 1 year

### Is this timing related to a specific event or Congressionally mandated deadline?
  - [x] Yes (OGC is asking this be live by Sept, which requires usability testing in Aug)
  - [ ] No

### If you are conducting research, when is that starting? 
**Provide date if available:** TBD Aug 15th (ish)
  - [ ] Next week
  - [ ] This month
  - [x] Next month
  - [ ] This quarter
  - [ ] Next quarter
  - [ ] 6 months
  - [ ] 1 year

### Will this new product be released incrementally (for instance 25% of users initially)? 
  - [ ] Yes
  - [x] No (at this time the plan is for a full release)
 
**Note**: _If you checked yes, we’ll reach out to discuss details about the content in the react widget (we use these widgets to display entry points for new products to a certain percentage of users who visit our static pages)._

  
### When do you expect to launch your product to 100% of users?
  
*Please provide an estimated date so our team can create other relevant tickets.*
- **We expect to launch our product to 100% of users on:** September 2023
  
### Will this work be going through the Collaboration Cycle?
- [x] Yes
- [ ] No

### If no ...
- Skip to the _supporting artifacts_ section, below.
 
#### Please provide the link to your Collaboration Cycle ticket: 
- **Ticket (number)**: [60961](https://app.zenhub.com/workspaces/collaboration-cycle-reviews-61d89e9e5f4c120011a0ad37/issues/gh/department-of-veterans-affairs/va.gov-team/60961)
- **Collab. Cycle Ticket Opened (date)**: June 28, 2023
 
#### Please check all of the phases in the Collaboration Cycle you have completed: 
- [x] Design Intent
- [ ] Midpoint Review
- [ ] Staging
 
##### Design Intent
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [x] Already completed, no assistance needed.
- [ ] Unknown at this time.
##### Midpoint
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed.
- [ ] Unknown at this time.
##### Staging
- [ ] Accessibility
- [ ] Content
- [x] IA
- [ ] Translation
- [ ] Already completed, no assistance needed.
- [ ] Unknown at this time.
  
## Supporting artifacts
*Please provide links to supporting artifacts as available.*
- **Product outline**: [Product info](https://github.com/department-of-veterans-affairs/va.gov-team/tree/f180bf496a0b9b06b71770f20a4091a3b9bc7b04/products/identity/terms%20and%20conditions)
- **User flows**: [User flow in Figma](https://www.figma.com/proto/Fhw9P7r6pyaCGXRmolruoQ/Terms-of-Use?page-id=0%3A1&node-%5B=&node-id=58-976&starting-point-node-id=1%3A493&show-proto-sidebar=1)
- **Prototype or mockups**: [User flow in Figma](https://www.figma.com/proto/Fhw9P7r6pyaCGXRmolruoQ/Terms-of-Use?page-id=0%3A1&node-%5B=&node-id=58-976&starting-point-node-id=1%3A493&show-proto-sidebar=1)
- **If this is in Sketch, which version or specific page should we focus on?** N/A
- **Content source of truth**:
- **Research plan:** [Research Folder in GH](https://github.com/department-of-veterans-affairs/va.gov-team/tree/91db02e7caf27417583e829894628cb7b3753e50/products/identity/Research/2023-07%20Terms%20of%20Use)
- **Any epics/issues that may be helpful**: 
- **Link to testing accounts** (please do not paste them in this ticket): 

### Have you already worked with someone from OCTO leadership?
- [ ] Danielle
- [ ] Beth
- [ ] Mikki
- [ ] Martha

### How do you plan to work with us on this request?
- [ ] I’ll take this to Beth or Danielle’s office hours.
- [ ] I’d like to schedule time to talk about this.
- [x] Let’s work async and meet as needed.

 
### Accessibility Help in Slack
The [#accessibility-help channel](https://dsva.slack.com/archives/C8E985R32) in the DSVA Slack, is also available to you as a resource. Everyone is welcome to ask questions or get help from our a11y (accessibility) specialists. Tag `@Terry Nichols` to get a11y help asap.

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag `@Terry Nichols`.**

- If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
- If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)
 
 - If you are creating an [experimental design,](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) please [also contact the DST. ](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- If you are requesting [a11y support for user research,](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name) please complete this form, next.

**Comment by kristinoletmuskat** [comment]
Hi! I'm Kristin, one of the IAs helping with this. Could ya'll share the url for the TOU page? The one proceeding the sign-out notification. @asg5704 @claytonzook

**Comment by asg5704** [comment]
![UF - 4 (ToU Page)](https://github.com/department-of-veterans-affairs/va.gov-team/assets/67602137/84bb37ce-859d-4b2c-9014-e9b36d227ef8)

**Comment by kristinoletmuskat** [comment]
I followed up with Alexander on slack, but we could prob go with https://www.va.gov/terms-and-conditions/sign-out or https://www.va.gov/terms-and-conditions/non-consent for this url. @laurwill any preference about this language? Or do you think the language on this page will change?

**Comment by laurwill** [comment]
Hi @kristinoletmuskat , @claytonzook just mentioned that they're testing "decline" as the button content. We haven't used "consent" anywhere in this content. Would /decline make more sense here?

**Comment by kristinoletmuskat** [comment]
I think https://www.va.gov/terms-and-conditions/decline is best! Thanks @laurwill

**Comment by sara-amanda** [comment]
Hi, @kristinoletmuskat, I saw that you closed this ticket. We are going to reopen this Epic, due to the a11y work connected to it and future support during collab. cycle in ticket. Thanks 😄
